### PR TITLE
Update CITATION.cff

### LINKED ISSUE #1778: Update citation file
We should add something like: "If you cite ESBMC >= 7.0 please cite us as below."

https://github.com/esbmc/esbmc/blob/master/CITATION.cff

Here is our .bib file:

````
​
Lucas Cordeiro
​
@InProceedings{esbmc2024,
    author    = {Rafael Menezes and
                 Mohannad Aldughaim and
                 Bruno Farias and
                 Xianzhiyu Li and
                 Edoardo Manino and
                 Fedor Shmarov and
        Kunjian Song and
        Franz Brauße and
        Mikhail R. Gadelha and
        Norbert Tihanyi and
        Konstantin Korovin and
        Lucas C. Cordeiro},
    title     = {{ESBMC} 7.4: {H}arnessing the {P}ower of {I}ntervals},
    booktitle = {$30^{th}$ International Conference on Tools and Algorithms for the Construction and Analysis of Systems (TACAS'24)},
    series       = {Lecture Notes in Computer Science},
    volume       = {14572},
    pages     = {376–380},
    doi       = {[https://doi.org/10.1007/978-3-031-57256-2_24 [doi.org]](https://urldefense.com/v3/__https://doi.org/10.1007/978-3-031-57256-2_24__;!!PDiH4ENfjr2_Jw!F88aCXYGWd1dZ7nb9wYc_tGznRjeXG-Mm5Ob2eTmqKjVK7J5wcrYv3yOoJEvMamGjXIKM65kw-4wLdG4lKJ9Jq-42YCnURgRGTs$)},
    year      = {2024},
    publisher = {Springer}
````

### PR DESCRIPTION
**Pull request by fbrausse**
Closes #1778.

I've used the authors from the v7.4 TACAS tool paper. Not all authors have an ORCiD known to DBLP, those entries I left out.

### CODE REVIEW

**Review by lucasccordeiro — APPROVED** [review]

File: CITATION.cff [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
+      given-names: Fedor
+      orcid: https://orcid.org/0000-0002-3848-451X
+    - family-names: Song
+      given-names: Kunjian
+    - family-names: Brauße
+      given-names: Franz
+      orcid: https://orcid.org/0000-0002-2386-7489
     - family-names: Gadelha
-      given-names: Mikhail R.
+      given-names: "Mikhail R."
```
Comment (mikhailramalho) [inline CITATION.cff]:
```suggestion
      given-names: "Mikhail"
```

File: CITATION.cff [hunk 2]
```diff
@@ -1,26 +1,43 @@
...
+      orcid: https://orcid.org/0000-0003-0028-5440
+    - family-names: Shmarov
+      given-names: Fedor
+      orcid: https://orcid.org/0000-0002-3848-451X
+    - family-names: Song
+      given-names: Kunjian
+    - family-names: Brauße
+      given-names: Franz
+      orcid: https://orcid.org/0000-0002-2386-7489
     - family-names: Gadelha
```
Comment (mikhailramalho) [inline CITATION.cff]:
```suggestion
    - family-names: "R. Gadelha"
```

**Review by mikhailramalho — APPROVED** [review]
LGTM, just need a small change to my surname

File: CITATION.cff [hunk 3]
```diff
@@ -1,26 +1,43 @@
...
+message: "If you cite ESBMC >= 7.0 please cite us as below."
 authors:
+    - family-names: "Sá Menezes"
+      given-names: Rafael
+      orcid: https://orcid.org/0000-0002-6102-4343
+    - family-names: Aldughaim
+      given-names: Mohannad
+      orcid: https://orcid.org/0000-0003-1708-1399
+    - family-names: Farias
+      given-names: Bruno
```
Comment (brcfarias) [inline CITATION.cff:11]:
```suggestion
      given-names: Bruno
      orcid: https://orcid.org/0009-0002-5124-8607
```

File: CITATION.cff (hunk 2 above)
Comment (fbrausse) [reply CITATION.cff]: Done, sorry for not realizing.

**Review by tihanyin — APPROVED** [review]

**Review by emanino — APPROVED** [review]

File: CITATION.cff [hunk 4]
```diff
@@ -1,26 +1,43 @@
...
-      given-names: Mikhail R.
+    - family-names: "Sá Menezes"
+      given-names: Rafael
+      orcid: https://orcid.org/0000-0002-6102-4343
+    - family-names: Aldughaim
+      given-names: Mohannad
+      orcid: https://orcid.org/0000-0003-1708-1399
+    - family-names: Farias
+      given-names: Bruno
+    - family-names: Xianzhiyu
```
Comment (XLiZHI) [inline CITATION.cff]:
```suggestion
    - family-names: Li
```

File: CITATION.cff [hunk 5]
```diff
@@ -1,26 +1,43 @@
...
+      given-names: Li
```
Comment (XLiZHI) [inline CITATION.cff]:
```suggestion
      given-names: Xianzhiyu
```

**Review by XLiZHI — APPROVED** [review]
Just minor changes ;)

File: CITATION.cff (hunk 5 above)
Comment (fbrausse) [reply CITATION.cff]: I'm very sorry, I should have know better. Thanks.

File: CITATION.cff [hunk 6]
```diff
@@ -1,26 +1,43 @@
...
+    - family-names: Shmarov
+      given-names: Fedor
+      orcid: https://orcid.org/0000-0002-3848-451X
+    - family-names: Song
+      given-names: Kunjian
+    - family-names: Brauße
+      given-names: Franz
+      orcid: https://orcid.org/0000-0002-2386-7489
+    - family-names: "R. Gadelha"
+      given-names: "Mikhail"
```
Comment (mikhailramalho) [inline CITATION.cff]:
```suggestion
      given-names: Mikhail
```

**Review by shmarovfedor — APPROVED** [review]

**Review by rafaelsamenezes — APPROVED** [review]

**Review by brcfarias — APPROVED** [review]

**Review by Mohannad-Aldughaim — APPROVED** [review]

**Comment by lucasccordeiro** [comment]
Thanks for submitting this PR, @fbrausse.

**Pull request merged**